packages/reporting-server/README.md

```diff
@@ -1,3 +1,10 @@
+# Description
+
+This project is about a server that exposes two APIs, one for writing logs to a persistent storage and the other for generate reports. As the project's name says, the idea is to work as a report server. This report server is designed to receive data from different pods within a Kubernetes cluster. The data is received through a data collection tool called Fluentd. Because fluentD is used, this project uses a parser for that format. You can find the log formats [here](https://github.com/open-rmf/rmf-web/blob/main/packages/reporting-server/rest_server/__mocks__/raw_data.py).
+
+In the following image, we can observe how all the pods interact with the reporting server
+![reporting-server](https://user-images.githubusercontent.com/11761240/123880883-9147b800-d911-11eb-88e8-f771914089ae.png)
```

Comment:
Thanks for the diagram! I corrected a typo
```suggestion
![rmf-web kubernetes cluster diagram](https://user-images.githubusercontent.com/28668944/123916706-8b56e300-d9b4-11eb-990f-69e717f87b38.png)

```

Reply:
![rmf-web kubernetes cluster diagram](https://user-images.githubusercontent.com/28668944/123916839-ac1f3880-d9b4-11eb-8231-67c93875b582.png)